sound/soc/sof/intel/ext_manifest.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause) */
```

Comment:
NAK on the file name.
You already have sound/sof/ext_manifest.h, changed in the previous patch.

This has got to be "intel_ext_manifest.h" or something clearly identifying Intel and not the generic SOF stuff

Reply:
@plbossart it is already in intel/ directory, can it really be confused for being SOF-generic? I find it a bit confusing in general: in most cases I prefer to eliminate redundancy in file names, so, sound/soc/sof/core.c is clear, that it's the SOF core, not ASoC or even ALSA core.c. But we also have sound/soc/sof/sof-audio.[ch], sof-priv.[ch], sof-{acpi,of,pci}-dev.c which seem both inconsistent and redundant, but ok, we already have them, and I personally don't see that as important enough to merit renames. OTOH ASoC has all their .c files named as sound/soc/soc-*.c which does look redundant to me, but at least it's consistent :-) In short, personally I find group/function.c preferable over group/group-function.c and it does seem to me the prevailing variant. But maybe this is a special case...

Reply:
I don't like duplication in general either but note the number of "intel" occurrences in `#include "ext_manifest.h` is zero, could be a bit confusing.

Now you also have to pray that the `Makefile`s know what they're doing which is quite rare :-) OK, hopefully not rare in the kernel...

Many editors also show only the basename in their menus etc.

Reply:
I like unique file name personally as I manage all the files in project. But I already found multiple same file names in linux kernel directory. My understanding... if same file in `include directory`, there will be strong objection. linux kernel seems allow same file name in different `source directory` and doesn't like redundancy in file name. This file is in SOF `source directory`.
I don't have strong opinion. @plbossart what's your position?